src/store/inflight_activation.rs

```diff
@@ -463,8 +698,14 @@ impl InflightActivationStore {
     pub async fn delete_activation(&self, id: &str) -> Result<(), Error> {
         sqlx::query("DELETE FROM inflight_taskactivations WHERE id = $1")
             .bind(id)
-            .execute(&self.write_pool)
+            .execute(&self.meta_write_pool)
             .await?;
+
+        sqlx::query("DELETE FROM taskactivation_blobs WHERE id = $1")
+            .bind(id)
+            .execute(&self.blob_write_pool)
+            .await?;
```

Comment:
Does this need an `if self.is_new_write()` guard?

Reply:
It doesn't, the table will exist regardless of the guards. The guards just determine what is in the table.